### ISSUE TITLE
Performance, Accessibility, Privacy Kleiderordnung-Berlin

### ISSUE THREAD

**Issue by openmindculture** [openmindculture/kleiderordnung#15]
Kleiderordnung-Berlin should use a modern editor-friendly theme suited for a one page portfolio.
But the current evaluation theme + elementor template still has issues concerning performance, accessibility, and privacy, see InkThemes issue ##299226##

> A first test in common analyzer tools (I should have done that with your demo before, but forgot to do so) shows only 37/100 in Google Page Speed - although the server is just unoptimized shared hosting and no WP cache plugin configured yet, I still would have expected at least a yellow score. Securityheaders.io gives an F but that can be fixed via .htacces file later. But I also got 88 WCAG 2.0 issues in Axe accessibility audit. Last but not least, the webfonts are not hosted locally but loaded from Google font servers which is no longer legal due to EUropean legislation.

Update: in the meantime, I have optimized some basic WP + Elementor settings, upgraded the server to PHP 8.0, and activated W3Total Cache, so now we have 57/100 at PageSpeed Insights.

How am I supposed to overcome these issues?

- [ ] Is there a recommended configuration that we still have to apply?
  - [ ] configure colors that provide necessary contrast to comply with WCAG 2.0
  - [x] optimize basic WP + Elementor settings
- [ ] Any additional plugins to install?
  - [x] configure W3 Total Cache for caching, headers, and image optimization
- [x] code a child theme to unload and override the problematic parts?
  - [ ] unregister external resources loading like Google fonts
  - [ ] use local fonts instead, optimized for intended glyph ranges (Latin / Western Europe)
  - [ ] reduce icon fonts etc.
- [ ] configure additional performance and security headers using .htaccess only if necessary
- [x] upgrade server to PHP 8.0 (from 7.4) if supported by the theme
- [x] check for Elementor issues (ensure not using the buggy one, recommended version is 3.1.0, currently installed)

**Comment by openmindculture** [comment]
> We will report all of these issues to our developers and will get back to you.
> Meanwhile please share the screenshots of the information with us and edited videos.
> We hope that we will surely help you out ASAP.

- [x] sent material to the developers
  - axe-audit (json and screenshot)
  - network tab screenshots showing fonts.gstatic / fonts.googleapi requests
  - page speed insights (pdf and screenshot)
  - webpagetest (screenshot, requests waterfall, details as pdf)

**Comment by openmindculture** [comment]
> I would recommend you to once create a child theme as you are mentioning.

[github.com/openmindculture/wp-colorway-child](https://github.com/openmindculture/wp-colorway-child)

**Comment by openmindculture** [comment]
obsolete when not using the theme anymore
